.github/workflows/rpm-builder.yaml

```diff
@@ -0,0 +1,244 @@
+      uses: docker/build-push-action@v6
+      id: build
+      env:
+        DOCKER_BUILD_SUMMARY: false
+      with:
+        context: "{{defaultContext}}:rpm"
+        tags: ${{ steps.metadata.outputs.tags }}
+        labels: ${{ steps.metadata.outputs.labels }}
+        platforms: ${{ matrix.platform }}
+        secrets: |
+          "RHEL_USERNAME=${{ secrets.RHEL_USERNAME }}"
+          "RHEL_PASSWORD=${{ secrets.RHEL_PASSWORD }}"
+        build-args: |
+          base_image=${{ matrix.base_image }}
+        outputs: type=docker,dest=${{ runner.temp }}/${{ env.IMAGE_NAME }}-${{ matrix.os }}-${{ matrix.arch }}.tar
+
+    - name: Upload builder image as artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: ${{ env.IMAGE_NAME }}-${{ matrix.os }}-${{ matrix.arch }}
+        path: ${{ runner.temp }}/${{ env.IMAGE_NAME }}-${{ matrix.os }}-${{ matrix.arch }}.tar
+
+  push:
+    name: Push image to the repo
+    strategy:
+      matrix:
+        os:
+        - alma-10
+        - centos-10
+        - rhel-10
+        - rocky-10
+        - alma-9
+        - centos-9
+        - rhel-9
+        - rocky-9
+        - alma-8
+        - centos-8
+        - rhel-8
+        - rocky-8
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    needs:
+    - build
+    services:
+      registry:
+        image: registry:2
+        ports:
+        - 5000:5000
+    steps:
+    - name: Download artifact images
+      uses: actions/download-artifact@v4
+      with:
+        pattern: ${{ env.IMAGE_NAME }}-${{ matrix.os }}-*
+        path: ${{ runner.temp }}/tar-images
+        merge-multiple: true
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ env.IMAGE_REPO }}
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Pushing image to the local repo
+      run: |
+        set -o errexit -o nounset -o pipefail
+        jobj="[]"
+        for image_file in ${{ runner.temp }}/tar-images/*.tar; do
+            image_name=$(docker load --input ${image_file} | sed -e 's/Loaded image: //')
+            repo_image_name=$(echo localhost:5000/${image_name})
+            jobj=$(echo "${jobj}" | jq -c ". + [\"${repo_image_name}\"]")
+            docker image tag ${image_name} ${repo_image_name}
+            docker push ${repo_image_name}
+        done
+        eval_command=$(echo "${jobj}" | jq -r '["docker buildx imagetools create --tag localhost:5000/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}:${{ matrix.os }}"] + . | join(" ")')
+        eval "${eval_command}"
+
+    - name: Push image to GHCR
+      run: |
+        docker buildx imagetools create \
+            --tag ${{ env.IMAGE_REPO }}/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}:${{ matrix.os }} \
+            localhost:5000/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}:${{ matrix.os }}
+
+  cleanup:
```

Comment:
Please remove the cleanup part. That is done at 
https://github.com/kamailio/kamailio-docker/blob/master/.github/workflows/github-cleanup.yml